sxg_rs/src/acme/mod.rs

```diff
@@ -174,9 +180,10 @@ pub async fn continue_challenge_validation_and_get_certificate<F: Fetcher, S: Si
             serde_json::Map::new(),
         )
         .await?;
-    // Repeatly checks the challenge object while it is being processed by the server.
+    const POLLING_INTERVAL: std::time::Duration = std::time::Duration::from_secs(1);
+    // Repeatedly checks the challenge object while it is being processed by the server.
     loop {
-        tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+        tokio::time::sleep(POLLING_INTERVAL).await;
```

Comment:
You should implement an exponential backoff algorithm for the delay. This is what's done for amppaclager now .

Reply:
Done